game/src/sandbox/speed.rs

```diff
@@ -761,3 +772,40 @@ fn compare_count(after: usize, before: usize) -> String {
         format!("-{}", prettyprint_usize(before - after))
     }
 }
+
+fn build_jump_to_time_btn(target: Time, ctx: &EventCtx) -> Widget {
+    Btn::text_bg2(format!("Jump to {}", target.ampm_tostring()))
+        .build(ctx, "jump to time", hotkey(Key::Enter))
+        .named("jump to time")
+        .centered_horiz()
+        .margin_above(16)
+}
+
+fn build_jump_to_delay_button(halt_limit: Duration, ctx: &EventCtx) -> Widget {
+    Btn::text_bg2(format!("Jump to next {} delay", halt_limit))
+        .build(ctx, "jump to delay", hotkey(Key::D))
+        .named("jump to delay")
+        .centered_horiz()
+        .margin_above(16)
+}
+
+fn build_jump_to_delay_picker(halt_limit: Duration, ctx: &EventCtx) -> Widget {
+    // EZGUI TODO: it'd be nice if we could style the fg color for persistent splits but this needs
```

Comment:
Hmm. I think of these, I'd vote for the traditional builder pattern. Just curious, what color do you want to change here; the fg of the text? Maybe that's something that belongs on `Choice` -- do you want different colors for each option?

Reply:
I rant into this because I noticed that we tend to use the PersistentSplit as a sort of hybrid control which lets you both "select one of these options" and separately "submit the selected option" button.

Trying to follow that pattern, I thought the drop-down duration picker would double as a button to "submit" the selected duration and "jump to that delay".

Since we have a style that we apply to "modal" submit buttons, like this:

<img width="245" alt="Screen Shot 2020-08-28 at 1 27 54 PM" src="https://user-images.githubusercontent.com/217057/91612449-4a281980-e932-11ea-8dad-9d728c775fb6.png">

...I was thinking the drop-down+submit widget, "Jump to 5min delay", should be styled like a submit button too, but that turns out to be non-trivial, so I went with two separate widgets: one to select, a second styled submit button.